src/analytics/Prices.js

```diff
@@ -0,0 +1,82 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { Col, Row } from 'reactstrap';
+
+import ErrorBoundary from 'components/ErrorBoundary';
+import { compareStrings, SortableTable } from 'components/SortableTable';
+import { cardEtchedPrice, cardFoilPrice, cardNormalPrice, cardPrice, cardStatus } from 'utils/Card';
+
+const Prices = ({ cards }) => {
+  const getValue = (collection, allowedStatus, allowedFinish) => {
+    const allowedStatuses = [];
+    if (allowedStatus === 'Owned' || allowedStatus === 'Any') allowedStatuses.push('Ordered', 'Owned', 'Premium Owned');
+    if (allowedStatus === 'Not Owned' || allowedStatus === 'Any') allowedStatuses.push('Not Owned', 'Proxied');
+
+    let priceFn;
+    switch (allowedFinish) {
```

Comment:
This is OK, just preemptively flagging this as something that we might want to extract into a separate function in the future, should this functionality (getting the price of a card in a specified finish) become more widely used.